bundle/manifests/rsct-operator.clusterserviceversion.yaml

```diff
@@ -173,7 +173,7 @@ spec:
                 - --leader-elect
                 command:
                 - /manager
-                image: quay.io/powercloud/rsct-operator:0.0.1
+                image: ghcr.io/ocp-power-automation/rsct-operator:latest
```

Comment:
will have to how to install any specific version other than latest tag all the time, let's create a task and explore on this.

Reply:
sure